redmine/issue.rb

```diff
@@ -41,11 +41,17 @@ def pull_request
     field['value']
   end
 
-  def set_pull_request(url)
+  def set_pull_requests(url)
     @raw_data['issue']['custom_field_values'] = {'7' => url}
     self
   end
 
+  def add_pull_request(url)
+    current_pull_requests = @raw_data['issue']['custom_field_values']['7']
```

Comment:
The JSON has a different layout to this:

<pre>
{
    "issue": {
        "id": 2,
        "project": {
            "id": 1,
            "name": "Test"
        },
        "tracker": {
            "id": 1,
            "name": "Bug"
        },
        "status": {
            "id": 1,
            "name": "New"
        },
        "priority": {
            "id": 2,
            "name": "Normal"
        },
        "author": {
            "id": 1,
            "name": "Redmine Admin"
        },
        "subject": "No PR",
        "description": "",
        "start_date": "2015-10-30",
        "done_ratio": 0,
        "spent_hours": 0.0,
        "custom_fields": [
            {
                "id": 1,
                "name": "Pull request",
                "multiple": true,
                "value": [

                ]
            }
        ],
        "created_on": "2015-10-30T13:01:00Z",
        "updated_on": "2015-10-30T13:01:00Z"
    }
}
</pre>